app/script/extension/GiphyRepository.js

```diff
@@ -123,8 +122,9 @@ z.extension.GiphyRepository = class GiphyRepository {
     return this.giphy_service.get_search({
       limit: 100,
       offset: offset,
-      query: options.query,
-      sorting: options.sorting,
+      // eslint-disable-next-line id-length
+      q: options.query,
```

Comment:
Hehe. I asked the same question but actually the parameter on the Giphy API is called "q": https://developers.giphy.com/docs/#search-endpoint